bmq-sdk/src/test/java/com/bloomberg/bmq/it/SessionIT.java

```diff
@@ -3955,453 +3947,6 @@ public void openCloseMultipleSubqueuesTest() throws BMQException {
         }
     }
 
-    /**
-     * Ensure that users can open priority queue with a tier specified in uri.
-     *
-     * <ul>
-     *   <li>start broker
-     *   <li>create session
-     *   <li>start session
-     *   <li>create and open priority queue with tier for reading and writing
-     *   <li>create and open priority queue w/o tier for reading
-     *   <li>post and receive messages
-     *   <li>close queues
-     *   <li>create and open priority queue with tier for reading and writing
-     *   <li>create and open priority queue w/o tier for reading and writing
-     *   <li>post and receive messages
-     *   <li>close queues
-     *   <li>stop session
-     * </ul>
-     */
-    @Test
-    public void testPriorityQueueTierOverride() throws BMQException, IOException {
```

Comment:
Could you provide some details on that?
In C++ SDK we have similar ITs

Reply:
The thing is, previously we had tiers description as a field in one `domains.json`, on a broker side.

Now we have static and separate structure for domains/tiers.
Meaning, that we need to prepare json-configuration file for each default domain and each qualified domain used.
For example, if we use `bmq.test.mmap.priority` and `bmq.test.mmap.priority.~lcl-localhost`, we need to create both `bmq.test.mmap.priority.json` and `bmq.test.mmap.priority.~lcl-localhost.json` config files for broker.

But broker needs to know that these 2 domains are related, so it can route messages across 2 domains with similar queue names. To do so, we have a *redirect* variant in domain config. It's when one config points to another, making relation similar to symbolic link. In the example, we can configure relation like this:
`bmq.test.mmap.priority.json` -> `bmq.test.mmap.priority.~lcl-localhost.json`

The important thing here is that this resolving performed on a broker side. So SDK doesn't handle it at all. It needs to be tested, but not on a ext lang SDK side, but somewhere near the broker. Tests for SDK should test the SDK itself, not the broker.